### DISCUSSION TITLE
What does Renovate do when I revert a merged renovate Pull Request?

### DISCUSSION THREAD

**Discussion by srmadsen** [renovatebot/renovate#15577] [Request Help] [answered]
What does Renovate do when I revert a merged renovate Pull Request?

**Comment by rarkins** [comment, marked as answer]
It will detect again that updates are available, but then treat the original merged PR as blocking 
(assuming it wasn't an immortal PR to begin with). You should see it in your dependency dashboard issue but no get an MR for it unless you ask for until a newer version is available.
